ports/espressif/common-hal/_bleio/CharacteristicBuffer.c

```diff
@@ -69,7 +81,8 @@ void _common_hal_bleio_characteristic_buffer_construct(bleio_characteristic_buff
     bleio_characteristic_obj_t *characteristic,
     mp_float_t timeout,
     uint8_t *buffer, size_t buffer_size,
-    void *static_handler_entry) {
+    void *static_handler_entry,
+    bool watch_for_interrupt_char) {
```

Comment:
I don't see that `watch_for_interrupt_char` is saved in this `_construct()` routine. It's just ignored. Did you want to save it?

Reply:
Good catch. Thank you!